### PR TITLE
V1.9.0 stage

### PR DESCRIPTION
**Pull request by richardpen**
* Feature - Add Amazon CloudWatch Logs logging driver.
* Enhancement - Add functional test for telemetry and awslogs driver.
* Enhancement - Fix Squid functional test.
* Enhancement - Improve unit test for ACS handler when adding task and reconnection on DiscoverPollEndpoint.
* Bug - Fix README ECS_LOGFILE description.
* Bug - Fix ACS handler when acking blank message ids.
* Bug - Fixed memory leak in ACS payload handler.
* Bug - Fixed multiple deadlocks.

r? @samuelkarp @aaithal @juanrhenals

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # Changelog
 
+## 1.9.0
+* Feature - Add Amazon CloudWatch Logs logging driver.
+* Bug - Fixed ACS handler when acking blank message ids.
+* Bug - Fixed incorrect reporting of CPU utilization.
```
Comment (samuelkarp) [inline CHANGELOG.md]: Can you rephrase these?

* Bug - Fixed an issue where CPU utilization could be reported incorrectly.
* Bug - Resolved a bug where containers would not get cleaned up in some cases.

**Comment by richardpen** [comment]
Rephrased the changelog

**Comment by aaithal** [comment]
:shipit:

**Comment by samuelkarp** [comment]
:+1:
